modules/admin/admin-ui/src/main/resources/web/admin/common/js/liveedit/ContentBackedComponentView.ts

```diff
@@ -0,0 +1,59 @@
+module api.liveedit {
+
+    import Component = api.content.page.region.Component;
+    import ContentSummary = api.content.ContentSummary;
+    import ContentSummaryBuilder = api.content.ContentSummaryBuilder;
+    import ContentSummaryAndCompareStatus = api.content.ContentSummaryAndCompareStatus;
+    import ContentTypeName = api.schema.content.ContentTypeName;
+
+    export class ContentBackedComponentViewBuilder<COMPONENT extends Component> extends ComponentViewBuilder<COMPONENT> {
+
+        contentTypeName: ContentTypeName;
+
+        constructor() {
+            super();
+        }
+
```

Comment:
Also, let's rename to ``ContentBasedComponentViewBuilder`` and ``ContentBasedComponentView``